compiler/lit.cfg.py

```diff
@@ -19,6 +19,9 @@
 config.suffixes = [".mlir", ".txt"]
 config.test_format = lit.formats.ShTest(execute_external=True)
 
+config.substitutions.append(("%lib_dir", config.lib_dir))
+config.substitutions.append(("%lib_ext", config.lib_ext))
```

Comment:
We could use static linking. But the problem is still the same. How to locate the file containing the plugin. The problem is that the file I need to send it via the RUN command, but I did not see a way to refer to the file location in the build directory. Maybe there is an easy way but I do not know what it is

Reply:
Also, thanks for the other pointers. I was looking inside of the build folder without realizing it. I see the scripts now.

Reply:
The idea of plugin/instrumentation passes are generally for those passes to be compiled out of the main compiler build and then just included at run/JIT compile time. So building the test pass as a static library defeats the purpose of the test in my opinion (i.e. to test the way a user would actually us it).

Reply:
Gotcha, thanks for clarifying.

I wonder if we can use this existing approach instead of modifying `lit.cfg.py`? https://github.com/iree-org/iree/blob/d5839584547ec2a258b12399f56ea3d006668dbb/samples/custom_module/dynamic/test/example.mlir#L1-L7

Reply:
Marking as resolved, will continue the discussion below.